aether-utils/aether-client/aether/client_test/__init__.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2018 by eHealth Africa : http://www.eHealthAfrica.org
```

Comment:
Why not `aether/client/test.py`? 
With this the previous file `aether/client.py` turns into `aether/client/__init__.py`

But we keep the same `aether/{module}/...` structure in all the packages/modules